app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,11 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
+    <style name="AppTheme" parent="Theme.Material3.DayNight.NoActionBar">
+        <item name="android:statusBarColor">@android:color/transparent</item>
+        <item name="android:navigationBarColor">@android:color/transparent</item>
+        <item name="android:windowBackground">@color/colorBackground</item>
+        <item name="android:windowLightStatusBar">@bool/windowLightStatusBar</item>
```

Comment:
Regarding [this](https://github.com/nimblehq/jetpack-compose-crypto/pull/30/files#r966613011) I think we can remove these 👇 
```suggestion
        <item name="android:navigationBarColor">@android:color/transparent</item>
        <item name="android:windowBackground">@color/colorBackground</item>
        <item name="android:windowLightStatusBar">@bool/windowLightStatusBar</item>
```

Reply:
@Wadeewee okay, so let's keep it then https://github.com/nimblehq/jetpack-compose-crypto/pull/30#discussion_r966616541